run_designs.py

```diff
@@ -249,7 +249,7 @@ def run_design(designs_queue):
                 "-overwrite",
                 "-no_save",
                 "-run_hooks",
-            ] + (["-verbose", "99"] if show_log_output else [])
+            ] + (["-verbose", "1"] if show_log_output else [])
```

Comment:
Perhaps an future enhancement is exposing this at the cli level

Reply:
sure whatever